### DISCUSSION TITLE
Contenu custom dans l'affiche

### DISCUSSION THREAD

**Discussion by alemangui** [betagouv/ma-cantine#985] [Retours utilisateur·rice]
Suite à un retour de Pauline, un usager voudrait ajouter des information supplémentaires à l'affiche générée par la plateforme. Iel a suggéré de rendre l'affiche modifiable.

**Comment by alemangui** [comment]
Si le besoin est d'ajouter un peu de customisation à l'affiche, je serai plus de l'idée de mettre un champ de texte libre qui sera inclus dans le rendu final, au lieu de rendre le fichier complètement modifiable. On pourrait creuser avec l'usager en question.

**Reply by hfroot** [reply]
Je pense ça aussi, mais il n'y a pas beaucoup d'espace alors on doit penser attentivement comment rendre l'affiche beau n'importe si il y a du texte libre ou non

**Reply by alemangui** [reply]
Yep, tu as raison. Et limiter le champ en nombre de caractères etc

**Comment by hfroot** [comment]
C'est en staging - je vois pas un façon de 'fermer' cette discussion sans la supprimer alors j'ai ajouté un label.
